script/Arcueid/Day 4/04_00_ARC04_6_2B.txt

```diff
@@ -472,7 +472,7 @@ So yeah, there actually do exist people who've made it their sole purpose to kil
 -- Page 504, Offset 6950.
 -- 「とくに陰湿で執念深いのが教会……西欧を中心にした宗教の異端狩り。主の意思を人の手で代行する、なんてお題目から代行者と呼ばれているわ。
 // executioner punnery near the end <-- good practice in general, these kind of sentences with 代行 as related to 代行者 are plenty in number (or at least, they were in MBTL...)
-"Most insidious and tenacious of all when it comes to this is the Church... heresy hunters from a religion based in Western Europe. They claim their purpose is to Execute the Lord's will by human hand, so naturally, they're called Executioners.
+"Most insidious and tenacious of all when it comes to this is the Church... heresy hunters from a religion based in Western Europe. They claim their purpose is to execute the Lord's will by human hands, so naturally, they're called Executioners.
```

Comment:
keep the capital Execute for this case, since its sort of a snide introduction to Executioners.

Reply:
maybe an italic instead of the caps?